app/locales/it/messages.po

```diff
@@ -892,9 +892,14 @@ msgstr "È possibile condividere questa visualizzazione copiando l'URL o incorpo
 
 #: app/components/form.tsx
 #: app/components/hint.tsx
+#: app/components/hint.tsx
 msgid "hint.loading.data"
 msgstr "Caricamento dei dati..."
 
+#: app/components/hint.tsx
+msgid "hint.loading.data.large.datasets"
+msgstr "Può richiedere più di 30 secondi per grandi insiemi di dati."
```

Comment:
@MichelaBurzio ? Could you take a quick look 😊 ? The original sentence is "It may take more than 30 seconds for large datasets.".